### ISSUE TITLE
[Feature]: Add consumet api provider

### ISSUE THREAD

**Issue by ld3z** [movie-web/movie-web#284]
### What feature do you want to add?

Add consumet api as a provider for anime

### Why do you want to have this feature?

Has lots of areas that it gets it’s data from including 9anime and crunchyroll making it perfect for watching almost everything you could think of

### Anything other details to share?

Not sure if it’s possible to implement but thought it could be something to consider 😅

project page: https://github.com/consumet/api.consumet.org

**Comment by lem6ns** [comment]
consumet is already added thru the flixhq scraper but maybe support for anime providers can be added

**Comment by frost768** [comment]
Even though you cannot choose it explicitly on the search page you can still find animes. Just choose series

**Comment by ld3z** [comment]
You're right, my mistake. 😅

But I've noticed that when searching for anime it's a little weird. For example I've searched for Darling in the Franxx and when clicked on to watch it plays normally and even plays dubbed. But when watching something like My Dress Up Darling it shows a completely different show than what it's supposed to be. I've also searched for a show like Sword Art Online which has both sub and dub of it, it only plays sub.
